### PR TITLE
DT-840: Temporarily disable token verification everywhere

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/api/tokenVerificationApi.js [hunk 1]
```diff
@@ -1,27 +1,16 @@
 const config = require('../config')
 const log = require('../log')
 
-const processError = error => {
```
Comment (petergphillips) [inline backend/api/tokenVerificationApi.js:4]: removed as not used

File: backend/api/tokenVerificationApi.js [hunk 2]
```diff
@@ -1,27 +1,16 @@
...
-  return false
-}
-
 const tokenVerificationApiFactory = client => {
   const verifyToken = context => {
     if (!config.apis.tokenverification.enabled) {
       log.debug('Token verification disabled, returning token is valid')
       return true
     }
-    log.debug(`Token verification enabled with context`)
```
Comment (petergphillips) [inline backend/api/tokenVerificationApi.js:17]: remove excess debugging

File: helm_deploy/values-preprod.yaml [hunk 3]
```diff
@@ -34,7 +34,7 @@ env:
   USE_OF_FORCE_URL: "https://preprod.use-of-force.service.justice.gov.uk"
   USE_OF_FORCE_PRISONS: "WRI,BLI,RSI"
   TOKENVERIFICATION_API_URL: "https://token-verification-api-preprod.prison.service.justice.gov.uk"
-  TOKENVERIFICATION_API_ENABLED: "false"
```
Comment (petergphillips) [inline helm_deploy/values-preprod.yaml:37]: since it is a string it evaluates to true :facepalm:

File: backend/api/tokenVerificationApi.js [hunk 4]
```diff
@@ -1,27 +1,16 @@
...
-    log.debug(context)
     return client
       .post(context, `/token/verify`)
       .then(response => {
-        const result = (response.body && response.body.active) || false
-        log.debug(`Token verification result is ${result}`)
-        return result
+        return (response.body && response.body.active) || false
```
Comment (JamieDraperUK) [inline backend/api/tokenVerificationApi.js:13]: Might be cleaner to use Boolean(response.body && response.body.active)

Comment (JamieDraperUK) [reply backend/api/tokenVerificationApi.js:13]: `.then(response =>  Boolean(response.body && response.body.active))`

Comment (petergphillips) [reply backend/api/tokenVerificationApi.js:13]: good suggestion, changed

**Pull request merged**